lib/cypress/patient_zipper.rb

```diff
@@ -22,8 +22,10 @@ def export(patient)
   end
 
   class QRDAExporter
-    C3EXPORTER = HealthDataStandards::Export::Cat1.new('r3')
-    C3_1EXPORTER = HealthDataStandards::Export::Cat1.new('r3_1')
+    C3EXPORTER = GoCDATools::Export::GoExporter.instance
+    C3_1EXPORTER = GoCDATools::Export::GoExporter.instance
```

Comment:
I think we actually do need them to be 3 different instances of the exporter. not positive but I think we should leave it like this for now. And I think Matt's future performance improvements will make this more efficient.